documentation/assemblies/assembly-kafka-connect-replicas.adoc

```diff
@@ -11,7 +11,7 @@
 
 = Replicas
 
-Kafka Connect clusters can run multiple of nodes.
+Kafka Connect clusters can consist of one or several nodes.
```

Comment:
I don't know why, but the _one or several_ sounds a bit strange to me. Would _one or more_ be better?

```suggestion
Kafka Connect clusters can consist of one or more nodes.
```